app/views/widgets/thermometer/_display.slim

```diff
@@ -0,0 +1,6 @@
+div.progress
+  / get a percentile value of width for the progress bar from count and goal
+  - @width = 100 * options['count'].to_f / options['goal'].to_f
+  div.progress-bar role="progressbar" aria-valuenow=options['count'] aria-valuemin="0" aria-valuemax=options['goal'] style="width: #{@width}%"
```

Comment:
I think `aria-valuemax` here should be 100?

Otherwise, if the max value is the `goal`, it seems like it's unnecessary to do the division above? I think either way works just fine, it's just that in this case I think you're mixing approaches. Am I misreading?